applications/MappingApplication/custom_mappers/interpolative_mapper_base.h

```diff
@@ -21,11 +21,16 @@
 // External includes
 
 // Project includes
+#include "containers/system_vector.h"
+#include "containers/distributed_system_vector.h"
```

Comment:
a problem is that here you are importing mpi, even in serial mode,

so what will happen if you compile this without mpi?

Reply:
now of your new classes uses MPI directly (which is what makes development much easier)

Reply:
correct, however here you are importing DistributedVector, which internally uses mpi...and you are importing it even with USE_MPI=OFF

Reply:
I checked, you don't call any MPI-methods directly. You call only methods of the `DataCommunicator` which are safe to be called (even though it doesn't make sense)

It compiles without MPI both locally and in the Win-CI